src/webob/acceptparse.py

```diff
@@ -771,6 +791,8 @@ def acceptable_offers(self, offers):
         This uses the matching rules described in :rfc:`RFC 7231, section 5.3.2
         <7231#section-5.3.2>`.
 
+        Any offers that do not match the media type grammar will be ignored.
```

Comment:
grammar feels weird to me in this context, but I don't know what would be better here!

Reply:
well steve didn't like abnf, you guys can duke it out but I'm fine with it the way it is.